CONTRIBUTING.rst

```diff
@@ -43,83 +54,72 @@ Using a Development Checkout
 ----------------------------
 
 You’ll have to create a development environment to hack on
-`google-endpoints-api-management`, using a Git checkout:
+`endpoints-management-python`_, using a Git checkout:
 
--   While logged into your GitHub account, navigate to the `google-endpoints-api-management`
-    [repo][1] on GitHub.
--   Fork and clone the `google-endpoints-api-management` repository to your GitHub account
+-   While logged into your GitHub account, navigate to the `endpoints-management-python repo`_ on GitHub.
+-   Fork and clone the `endpoints-management-python` repository to your GitHub account
     by clicking the "Fork" button.
--   Clone your fork of `google-endpoints-api-management` from your GitHub account to your
+-   Clone your fork of `endpoints-management-python` from your GitHub account to your
     local computer, substituting your account username and specifying
-    the destination as `hack-on-google-endpoints-api-management`. For example:
-
-    ```bash
-    $ cd ${HOME}
-    $ git clone git@github.com:USERNAME/google-endpoints-api-management.git hack-on-google-endpoints-api-management
-    $ cd hack-on-google-endpoints-api-management
-    $ # Configure remotes such that you can pull changes from the google-endpoints-api-management
-    $ # repository into your local repository.
-    $ git remote add upstream https://github.com:google/google-endpoints-api-management
-    $ # fetch and merge changes from upstream into master
-    $ git fetch upstream
-    $ git merge upstream/master
-    ```
+    the destination as `hack-on-endpoints-management-python`. For example:
+
+  .. code:: bash
+
+    cd ${HOME}
+    git clone git@github.com:USERNAME/endpoints-management-python.git hack-on-endpoints-management-python
+    cd hack-on-endpoints-management-python
+
+    # Configure remotes such that you can pull changes from the endpoints-management-python
+    # repository into your local repository.
+    git remote add upstream https://github.com:google/endpoints-management-python
+
+    # fetch and merge changes from upstream into master
+    git fetch upstream
+    git merge upstream/master
```

Comment:
I'm wondering whether we can remove all the details and just refer readers to https://guides.github.com/introduction/flow/